### DISCUSSION TITLE
v0.41.2-20220628.220813

### DISCUSSION THREAD

**Discussion by qvacua** [qvacua/vimr#964] [General]
* Bugfix: Sometimes macOS returns a weird locale, e.g., `en_DE.UTF-8`, which Neovim does not understand.

<hr /><em>This discussion was created from the release <a href='https://github.com/qvacua/vimr/releases/tag/v0.41.2-20220628.220813'>v0.41.2-20220628.220813</a>.</em>

**Comment by polydevuk** [comment]
As far as I can remember the NVIM version strings prior to this were simply v0.7.1, v0.7.2 etc. but this one is v.0.7.3-dev+nvimserver-0.7.0-1-11-geddb2ab75. This version is also much slower to start and I experience a beachball on startup with no change to my config. What is this "nvimserver"? I can't find any reference to it on Google other than neovim-server which is deprecated.
